.github/workflows/ci.yml

```diff
@@ -76,11 +77,12 @@ jobs:
           cargo test --package mbtiles --no-default-features
           cargo test --package mbtiles
           cargo test --package martin
-          cargo test --doc
+          cargo test --package martin-core
```

Comment:
what about doc tests?

Reply:
What about them?
They are in the new section below because they are kind of different and imo fail differently.

If we merge all the unit tests into one just command they should be included. Currently this makes more sense to me  being split

Reply:
ah, thx, didn't see it